### ISSUE TITLE
missing config.w32

### ISSUE THREAD

**Issue by ricardok1** [derickr/quickhash#3]
Hi Derick,

Can you please provide the config.w32 for being to compile on Windows.
I think this standard C so no problem on Windows, correct?

Thanks
Ricardo

**Comment by cmb69** [comment]
> I think this standard C so no problem on Windows, correct?

Not quite. The code casts to union types, which is, to my knowledge, not standard C, but a GCC extension. That is definitely not supported by MSVC.

There are some other (minor?) issues as well.
